examples/tree_2d_fdsbp/elixir_advection_extended.jl

```diff
@@ -8,8 +8,8 @@ using Trixi
 ###############################################################################
 # semidiscretization of the linear advection equation
 
-advectionvelocity = (0.2, -0.3)
-equations = LinearScalarAdvectionEquation2D(advectionvelocity)
+advection_velocity = (0.2, -0.3)
```

Comment:
Why not
```suggestion
advection_velocity = (0.2, -0.7)
```
?

Reply:
Oh, I missed this one. That change is just from the "replace all".